### DISCUSSION TITLE
How to upload multiple files to one index, chat with multiple files.

### DISCUSSION THREAD

**Discussion by alanwnl** [guangzhengli/ChatFiles#49] [Q&A] [answered]
How to upload multiple files to one index, chat with multiple files?

**Comment by guangzhengli** [comment, marked as answer]
You can compress files and then upload the compressed package. It is important to note that currently recursive file searching is not supported, so folders cannot exist within the compressed package.

**Reply by Sagar27888** [reply]
So if i want to upload multiple files we need to add in one folder and comprase it and then upload it?  Correct?

**Reply by guangzhengli** [reply]
Currently only supports selecting multiple files for compression, rather than the way to compress a folder, and it will be optimized asap.

**Reply by alanwnl** [reply]
Uploaded the compressed file, but how to reference that file?

<img width="1412" alt="Screenshot 2023-04-25 at 9 58 17 PM" src="https://user-images.githubusercontent.com/885966/234300086-de3a50b9-e169-4a63-afbc-9ab02c73688c.png">
